### PR TITLE
artifacts: add file size to files in artifacts view

### PR DESCRIPTION
**Pull request by obiwac**
Something that has always slightly annoyed me with the artifacts view 😄 

Not in a position to test any of this, so this is more of a feature request. Will try this out correctly this evening.

### CODE REVIEW

**Comment by obiwac** [comment]
Set up the web interface on my machine and everything looks fine as far as I can tell. Also made the sizes human-readable.

File: src/components/artifacts/ArtifactsView.tsx [hunk 1]
```diff
@@ -168,7 +191,7 @@ function ArtifactsView(props: Props) {
             <ListItemIcon>
               <InsertDriveFile />
             </ListItemIcon>
-            <ListItemText primary={info.path} />
+            <ListItemText primary={info.path} secondary={info.human_size} />
```
Comment (fkorotkov) [inline src/components/artifacts/ArtifactsView.tsx:194]: Probably can do `bytesToHumanReadable(info.size)` right away without a need for `human_size`. What do you think?

Comment (obiwac) [reply src/components/artifacts/ArtifactsView.tsx:194]: That would make more sense yeah. Web dev isn't really my area of expertise, so pardon if there are a few things that are usually done in better ways.

Comment (fkorotkov) [reply src/components/artifacts/ArtifactsView.tsx]: Not my cup of tea as well. 😅  No worries and thank you for the contribution!

**Review by fkorotkov — APPROVED** [review]

File: src/components/artifacts/ArtifactsView.tsx (hunk 1 above)
Comment (obiwac) [reply src/components/artifacts/ArtifactsView.tsx]: My pleasure!

**Pull request merged**